strictdoc/export/rst/templates/requirement.jinja.rst

```diff
@@ -6,25 +6,38 @@
 {{ _print_rst_header(requirement.title, requirement.ng_level) }}
 {% endif %}
 
-{%- if requirement.uid is not none %}
+{%- if requirement.has_meta > 0 %}
 .. list-table::
     :align: left
     :header-rows: 0
+    {% for meta_field in requirement.enumerate_meta_fields(skip_multi_lines=True) %}
+    * - **{{meta_field[0]}}:**
+      - {{ meta_field[1] }}
+    {% endfor %}
 
-    * - **UID:**
-      - {{requirement.uid}}
-{% endif %}
-
+{%- endif %}
 {%- if requirement.statement is not none %}
 {{requirement.statement}}
 {% elif requirement.statement_multiline is not none %}
 {{requirement.statement_multiline}}
 {% endif %}
 
+{%- if requirement.rationale or requirement.rationale_multiline -%}
+    **Rationale:** {{ renderer.render_requirement_rationale(requirement) }}
```

Comment:
I haven't checked the rendered PDF for this, but are you sure you want Rationale to be treated like a quote block?

Reply:
I mean, shouldn't it start like `Comment:` from the beginning of the line?

Reply:
I'll change it to a normal paragraph, no worries